packages/component/src/SendBox/MicrophoneButton.js

```diff
@@ -48,19 +58,30 @@ export default connectMicrophoneButton(
     language,
     styleSet
   }) =>
-    <div className={ classNames(
-      styleSet.microphoneButton + '',
-      ROOT_CSS + '',
-      (className || '') + '',
-      { dictating }
-    ) }>
+    <div
+      aria-controls="sr-Microphone"
+      className={ classNames(
+        styleSet.microphoneButton + '',
+        ROOT_CSS + '',
+        (className || '') + '',
+        { dictating }
+      ) }
+    >
       <IconButton
         alt={ localize('Speak', language) }
         disabled={ disabled }
         onClick={ click }
       >
         <MicrophoneIcon />
       </IconButton>
+      <div 
+        aria-live="polite"
+        className="sr-only"
+        id="sr-Microphone"
+        tabIndex="-1"
```

Comment:
Do we need `tabIndex`? I think `<div>` is no tabbable.

If we need, prefer `tabIndex={ -1 }`.

Reply:
You're right, no `tabIndex` necessary